src/assets.ts

```diff
@@ -9,6 +9,10 @@ export function setAssetContext(context: ExtensionContext) {
     EXTENSION_CONTEXT = context;
 }
 
+export function getAssetContext(): ExtensionContext | null {
+    return EXTENSION_CONTEXT;
```

Comment:
💡 observation: what instance of this returns `null` given the return type is `ExtensionContext | null`

Reply:
not entirely sure when this would be null, just referenced return type from line 6 of the file. But, if this is null, we handle it in recentCluster.ts